src/Tools/dotnet-dev-certs/src/Program.cs

```diff
@@ -110,6 +114,10 @@ public static int Main(string[] args)
                     "Display warnings and errors only.",
                     CommandOptionType.NoValue);
 
+                var checkJsonOutput = c.Option("--check-json-output",
```

Comment:
This name isn't great... it sounds like it's checking some JSON.

Since we only expect this to be used by tools it feels like a more verbose name is okay... maybe `--check-trust-machine-readable` or something?

Reply:
I agree the name wasn't great, I changed it to use your suggestion.